client/android/app/build.gradle

```diff
@@ -25,7 +25,7 @@ if (flutterVersionName == null) {
 android {
     namespace "com.appveyor.flet"
     compileSdkVersion flutter.compileSdkVersion
-    ndkVersion flutter.ndkVersion
+    ndkVersion "25.1.8937393"
```

Comment:
Is it accidental change or this is required by Geolocation plugin?

Reply:
It's required by flutter while testing on android device